docs/RELEASE_POLICY.md

```diff
@@ -86,7 +86,7 @@ also be specifying the supported kernel versions.
 
 | Release | Release Date | Latest Patch | Min. end of support | Official end of Support        |
 | ------: | -----------: | -----------: | ------------------: | :----------------------------- |
-| v1.3    |   2023-03-02 | v1.3.0       |          2023-09-02 | Supported                      |
+| v1.3    |   2023-03-02 | v1.3.1       |          2023-09-06 | Supported                      |
```

Comment:
does a patch release extend the support period?

Reply:
hm, very good question

Reply:
let's keep the old date...

Reply:
done